__tests__/app.test.js

```diff
@@ -111,4 +111,40 @@ describe('/api/articles', () => {
           })
       })
   })
-})
\ No newline at end of file
+})
+describe('/api/articles/:article_id/comments', () => {
+  test('GET: 200 sends array of all comments from a single article to the client', () => {
+      return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(2);
+        body.comments.forEach((comment) => {
+          expect(comment.article_id).toBe(3);
+          expect(typeof comment.comment_id).toBe('number');
+          expect(typeof comment.votes).toBe('number');
+          expect(typeof comment.created_at).toBe('string');
+          expect(typeof comment.author).toBe('string');
+          expect(typeof comment.body).toBe('string');
```

Comment:
Checking the types is valid, however, there's a lot of them! Have a look at ```toMatchObject``` to help with this